public/api/deleteFile.php

```diff
@@ -28,21 +28,21 @@
 }
 //TODO: Do we need canEdit permission to delete?
 
-//Test if user has permission to delete files
-$canUpload = FALSE;
-$sql = "
-SELECT canUpload 
-FROM user 
-WHERE userId = '$userId'
-";
-$result = $conn->query($sql);
-$row = $result->fetch_assoc();
-if ($row['canUpload'] == '1'){$canUpload = TRUE;}
-
-if (!$canUpload){
-  $success = false;
-  $message = "You don't have permission to delete files.";
-}
+// //Test if user has permission to delete files
```

Comment:
If we don't need this code can we just delete it? Also do we want to just rip off the band aid and remove the database field? We need to get more impactful database modifications worked into our development/deployment process at some point.

Reply:
Removed commented out code.  We should remove the database field.  We should clean the database as a combined task going forward.